ldj-readme-renderer/render.js

```diff
@@ -8,45 +8,57 @@
 const fs = require('node:fs');
 const process = require('node:process');
 
-const OUTPUT = `README.md`;
+const OUTPUT = `../README.md`;
 
-const TEMPLATE = `[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=coverage)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
-[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_index)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
-[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=reliability_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
-[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
-# Living Documentation Java
-
-This project is an extension of the [Living Documentation][ldoc] set of tools to
-Java. As part of the Living Documentation toolchain, it can be used to generate
-documentation and diagrams from source code so that documentation is always
-up-to-date.
-
-## Analyzer
-
-The analyzer parses Java projects into Living Documentation JSON files that can
-be rendered by Living Documentation renderers. It accepts the following command
-line options:
+const json = JSON.parse(fs.readFileSync(process.argv[2]));
 
+// Section for rendering the command line options
+const CMD_LINE_TABLE_TEMPLATE = `
```

Comment:
Why is this here, and not in `README_template.md`?

Reply:
Because the Javascript code filling the table is dependent on the exact form of this smaller template (i.e. #columns & headers)